### PR TITLE
functional tests added and updated utils

### PR DESCRIPTION
**Pull request by swapnaChetty**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-785

### Change description ###
Added new functional tests for forms controller and updated some utils.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/IntegrationTestBase.java [hunk 1]
```diff
@@ -20,25 +19,22 @@
...
 
     @Autowired
     public void initialise(@Value("${probate.orchestrator.service.url}") String orchestratorUrl,
-                           @Value("${probate.submit.url}") String submitServiceUrl,
                            @Value("${user.auth.provider.oauth2.url}") String idamUrl) {
         RestAssured.baseURI = orchestratorUrl;
         RestAssured.defaultParser = Parser.JSON;
         this.orchestratorUrl = orchestratorUrl;
-        this.submitServiceUrl = submitServiceUrl;
         this.idamUrl = idamUrl;
```
Comment (meganaslattery) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/IntegrationTestBase.java:30]: Are the orchestratorUrl or idamUrl used anywhere else in the code? do they need to be defined here?

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/FormsFunctionalTests.java [hunk 2]
```diff
@@ -2,146 +2,256 @@
...
     }
 
     @Test
-    @Pending
-    public void shouldCreateDraftThenSubmitAndFinallyUpdatePayment() throws IOException {
-        shouldSaveFormSuccessfully();
-        shouldUpdateForm();
-        shouldSubmitForm();
-        shouldUpdatePaymentSuccessfully();
+    public void shouldCreateDraftThenSubmitAndFinallyUpdatePayment() throws IOException, JSONException {
```
Comment (meganaslattery) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/FormsFunctionalTests.java:140]: Should this test be here if it's commented out?

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/utils/TestUtils.java [hunk 3]
```diff
@@ -29,23 +54,21 @@ public String getJsonFromFile(String fileName) {
...
+    public Headers getCitizenHeaders() {
+        return getHeaders(citizenEmail);
     }
 
-    public Headers submitHeaders(String sessionId) {
-        return Headers.headers(
-            new Header("Content-Type", ContentType.JSON.toString()),
-            new Header("UserId", sessionId),
-            new Header("Authorization", "DUMMY_KEY"));
+    public Headers getCaseworkerHeaders() {
```
Comment (meganaslattery) [inline src/functionalTest/java/uk/gov/hmcts/probate/functional/utils/TestUtils.java:61]: Is this used?

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/FormsFunctionalTests.java (hunk 2 above)
Comment (swapnaChetty) [reply src/functionalTest/java/uk/gov/hmcts/probate/functional/tests/FormsFunctionalTests.java:140]: As this is existing broken test which I try to fixed it but looks like flaky test, need some more time to investigate and fix it. I will add @Pending tag to the test.

File: src/functionalTest/java/uk/gov/hmcts/probate/functional/utils/TestUtils.java (hunk 3 above)
Comment (swapnaChetty) [reply src/functionalTest/java/uk/gov/hmcts/probate/functional/utils/TestUtils.java:61]: currently not used but going forward for other request we will be using it.

**Review by meganaslattery — APPROVED** [review]

**Pull request merged**